index.html

```diff
@@ -296,4 +297,125 @@ <h2>About me</h2>
       crossorigin="anonymous"
     ></script>
   </body>
+
+  <footer id="contact">
+    <div class="footer-text">
+      <h2 class="title">Contact me</h2>
+      <p class="supporting-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+    </div>
+    <form
+      action="https://formspree.io/f/xwkyvenp"
+      method="get"
+      class="form"
+      id="form"
```

Comment:
- [ ] Great job so far with the form, however, is still an issue you need to address to have your project approved. the form `method` is supposed to be a `post` request, not a `get` request. please kindly fix that :wink: :wink: